imagepullsecret-patcher/.gitignore

```diff
@@ -0,0 +1,18 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+imagepullsecret-patcher
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+.vscode/
+.DS_Store
```

Comment:
This file seems unnecessary if this isn't a separate git module

Reply:
yep, i'll get rid of it.